cli/src/commands/start.js

```diff
@@ -2,6 +2,7 @@ const { reporter, chalk } = require('@dhis2/cli-helpers-engine')
 const detectPort = require('detect-port')
```

Comment:
Since `index.html` now links to a `manifest.json` file, there will be an error if `manifest.json` is missing. So the manifests are generated on the `start` script in addition to `build` to avoid that error.